### PR TITLE
monai 0.7 weekly version + docs theme change

### PR DESCRIPTION
**Pull request by SachidanandAlle**
Signed-off-by: Sachidanand Alle <sachidanand.alle@gmail.com>

- [x] Bump up to monai-weekly (monitor for any PR/Workflow fails)
- [x] Sync the sphinx theme with monai 
- [x] Fix API doc generation warnings

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 commonmark==0.9.1
 recommonmark==0.6.0
-Sphinx==3.3.0
-sphinx-rtd-theme==0.5.0
+Sphinx
```
Comment (aihsani) [inline docs/requirements.txt:5]: We should fixate all requirements (including below)

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 aiofiles==0.6.0
 fastapi==0.65.2
-monai[nibabel, skimage, pillow, tensorboard, gdown, ignite, torchvision, itk, tqdm, lmdb, psutil, openslide]>=0.6.0
+monai-weekly[nibabel, skimage, pillow, tensorboard, gdown, ignite, torchvision, itk, tqdm, lmdb, psutil, openslide]
```
Comment (aihsani) [inline requirements.txt:3]: Why monai weekly?

File: monailabel/tools/validate_tta.py [hunk 3]
```diff
@@ -16,88 +16,96 @@
 
 import requests
```
Comment (aihsani) [inline monailabel/tools/validate_tta.py:17]: not sure if this file should be here... this doesn't seem related to the items in the PR.

**Review by aihsani — CHANGES_REQUESTED** [review]

File: monailabel/tools/validate_tta.py (hunk 3 above)
Comment (SachidanandAlle) [reply monailabel/tools/validate_tta.py:17]: docstring warning... direct exit issue..

File: requirements.txt (hunk 2 above)
Comment (SachidanandAlle) [reply requirements.txt:3]: only early testing.. end of next we rollback to 0.7 stable version when monai is released

File: docs/requirements.txt (hunk 1 above)
Comment (SachidanandAlle) [reply docs/requirements.txt:5]: borrowing it from monai.. they have removed the same..  https://github.com/Project-MONAI/MONAI/pull/2657/files#diff-2c9c11d26b09b8afde329980309d967121543a456e4592c76886a20b5cf56c90

**Review by diazandr3s — APPROVED** [review]

File: docs/requirements.txt (hunk 1 above)
Comment (aihsani) [reply docs/requirements.txt:5]: I don't think this is a good idea. That includes for the lines below

**Review by aihsani — APPROVED** [review]
